### PR TITLE
add Notifications page

### PR DESCRIPTION
**Pull request by alexmatson-da**

### CODE REVIEW

File: ui2/src/pages/notifications/NotificationTypes.tsx [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+  | TradingServiceOffer
+  | MatchingServiceOffer
+  | ClearingServiceOffer
+  | IssuanceServiceOffer
+  | RegulatorServiceOffer
+  | SettlementServiceOffer
+  | MarketServiceOffer
+  | AuctionServiceOffer
+  | BiddingServiceOffer;
+
```
Comment (Sarahb-da) [inline ui2/src/pages/notifications/NotificationTypes.tsx:152]: oh wow

Comment (Sarahb-da) [reply ui2/src/pages/notifications/NotificationTypes.tsx:152]: but in a way its very satisfying haha

**Review by Sarahb-da — APPROVED** [review]
the union types are becoming quite common in this code base, I like the pattern of creating `__Types.tsx` to keep them all together but out of the way.

Looks good!

**Review by alexgraham-da — APPROVED** [review]
looks good to me! I know there's a lot of copy pasta in here but I like how you set this up 😃

**Pull request merged**